src/core/LanguageMenu/LanguageMenu.test.tsx

```diff
@@ -31,6 +31,9 @@ test(
       'aria-valid-attr-value': {
         enabled: false,
       },
+      region: {
```

Comment:
Same axe util things apply to this as above.

Reply:
This was fixed and region is now disabled by default and does not need to be set here as well.